test/services/editions/db/EditionsDBTest.scala

```diff
@@ -393,6 +393,30 @@ class EditionsDBTest extends FreeSpec with Matchers with EditionsDBService with
       specialFront2.isHidden shouldBe false
     }
 
+    "should delete an issue and cascade the deletion to related entities" taggedAs UsesDatabase in {
+      val issue = insertSkeletonIssue(2019, 9, 1,
+        front("news/uk",
+          collection("politics", Some(CapiPrefillQuery("magic-politics-query", PathType.PrintSent)),
+            article("12345"),
+            article("23456")
+          )
+        )
+      )
+
+      val dbIssue: EditionsIssue = editionsDB.getIssue(issue).value
+      val frontIds = dbIssue.fronts.map(_.id)
+      frontIds.length shouldBe 1
+
+      editionsDB.getFrontFromIssueId(dbIssue.id) shouldNot be
```

Comment:
This is a thing that you only use in a test right? In which case I'd move the SQL into the test file.

Reply:
Possibly inline...

Reply:
Updated now.